src/app/core/user/user-admin-service/keycloak-admin.service.ts

```diff
@@ -84,9 +84,13 @@ export class KeycloakAdminService extends UserAdminService {
     userEntityId: string,
   ): Observable<{ userDeleted: boolean }> {
     return this.getUser(userEntityId).pipe(
-      switchMap((userAccount) =>
-        this.http.delete(`${this.keycloakUrl}/users/${userAccount.id}`),
-      ),
+      switchMap((userAccount) => {
+        if (!userAccount) {
+          Logging.warn("User not found in Keycloak", { userEntityId });
```

Comment:
```suggestion
          Logging.debug("User not found in Keycloak", { userEntityId });
```
if this didn't throw special errors, it's expected behavior (not every entity that can have an account must have one also)